server/src/tenant-manager/tenant-manager.service.ts

```diff
@@ -126,6 +138,59 @@ export class TenantManagerService {
     standardObjectsPrefillData(workspaceDataSource, dataSourceMetadata.schema);
   }
 
+  public async injectWorkspaceData(
```

Comment:
Since both functions are not used yet, I'm not sure I have the full picture of this implementation. How do we plan to use this? Do we want to merge this PR with those unused functions?

Reply:
It is used in the migration script but GitHub does not show it you need to expand it

Reply:
Oh I missed that one, my bad! Thanks